src/main/java/pl/sda/eventmanager/services/UserValidator.java

```diff
@@ -0,0 +1,96 @@
+package pl.sda.eventmanager.services;
+
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.stereotype.Service;
+import org.springframework.validation.Errors;
+import org.springframework.validation.ValidationUtils;
+import org.springframework.validation.Validator;
+import pl.sda.eventmanager.dto.LoginForm;
+import pl.sda.eventmanager.dto.RegisterForm;
+import pl.sda.eventmanager.model.User;
+
+@Service
+public class UserValidator implements Validator {
+
+    private final UserService userService;
+
+    public UserValidator(UserService userService) {
+        this.userService = userService;
+
+    }
+
+
+    @Override
+    public boolean supports(Class<?> aClass) {
+        return User.class.equals(aClass);
+    }
+
+    @Override
+    public void validate(Object o, Errors errors) {
+        RegisterForm registerForm = (RegisterForm) o;
+
+        User user = User.UserBuilder
```

Comment:
no need to builder new `User` just for purpose of validation, validate `RegisterForm`